### ISSUE TITLE
Pro Presenter 6 support 

### ISSUE THREAD

**Issue by kevindoveton** [jswetzen/ppstagedisplay#9]
Add propresenter 6 [live view](http://www.renewedvision.com/propresenter.php?page=wn) support

**Comment by jswetzen** [comment]
It would be great to keep up to date with new versions. I'm not using ProPresenter much at the moment, so I don't know what it would take to support live view. But since you point it out, is is something you want to work on?

**Comment by kevindoveton** [comment]
I've started work on this in a seperate repository as they use websockets to communicate. Once I've made significant progress we can work on implementing the changes. 

https://github.com/kevindoveton/Pro6StageDisplay

**Comment by jswetzen** [comment]
That sounds like a big change, maybe a complete rewrite! Is it a client-only app then so that each client communicates directly with ProPresenter? Maybe a node.js server is needed for finding ProPresenter with bonjour.

**Comment by kevindoveton** [comment]
Yep the client can communicate directly with propresenter - interesting that to get the liveview feature you dont actually need to authenticate! You do still need to authenticate for the regular stage display however.

Yeah theoretically if the client new the ip + port of the computer they wouldn't need node at all and it could be completely client side. Bonjour definitely makes things much simpler though, and a node server is probably just as easy to set up as any other server.
